arena.js

```diff
@@ -0,0 +1,46 @@
+class Arena{
+  constructor(name){
+    this.name = name;
+    this.gladiator= [];
+  }
+
+  addGladiator(x){
```

Comment:
this is fine, but in general i'd recommend more semantic names for parameters other than `x`